app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+
+const { getTopics } = require("./controllers/topics.controller");
+
+const app = express();
+
+app.use(express.json());
```

Comment:
Do you need this for get requests?

Reply:
I don't need them for the get request , but I thought  I might need them when sending posting to our data so I just wrote them.